image.yaml

```diff
@@ -56,13 +56,13 @@ modules:
           - name: os-eap-probes
           - name: jboss-maven
           - name: os-eap-db-drivers
-          - name: os-eap-sso
           - name: os-eap70-sso
```

Comment:
BTW, @iankko is keeping os-eap70-sso here going to be a problem, or should it be moved too?

Reply:
AFAICT it can be moved too as far as it is placed right before the call to ```os-eap-extensions``` module (so the default adapters version are installed a bit sooner than the possibly overridden ones from the extension module). Want me to move ```os-eap70-sso``` below too?

Reply:
Ok, will move it then above the extensions module (so it still works for KEYCLOAK-7680 purpose).